src/components/screens/editor/editor-screen.tsx

```diff
@@ -53,7 +53,8 @@ const EditorScreen: React.FC<Props> = ({
   const [ leftToolBarIndex, setLeftToolBarIndex ] = React.useState(0);
   const [ rightToolBarIndex, setRightToolBarIndex ] = React.useState(0);
   const [ addingKnots, setAddingKnots ] = React.useState(false);
-  const [ dataChanged, setDataChanged ] = React.useState(false);
+  const [ dataChanged, /* setDataChanged */ ] = React.useState(false);
```

Comment:
Why this is commented out?

Reply:
To suppress CloudFlare error for unused constant. The setDataChanged constant will be needed later on when implementing the save button functionality.

Reply:
👍🏻